core/core.ex

```diff
@@ -48,7 +51,21 @@ defmodule AntikytheraCore do
     System.put_env("ERL_LIBS", new_value)
   end
 
-  defp establish_connections_to_other_nodes(tries_remaining \\ 3) do
+  @connection_retrial_interval_in_milliseconds 5_000
+
+  defunpt calculate_connection_trial_count_from_health_check_grace_period() :: pos_integer do
+    connection_retrial_interval_in_seconds = @connection_retrial_interval_in_milliseconds / 1000
+
+    result =
+      trunc(
+        ClusterConfiguration.health_check_grace_period_in_seconds() /
+          connection_retrial_interval_in_seconds
+      )
+
+    if result < 1, do: 1, else: result
```

Comment:
It could be written as simply as following.

```suggestion
    max(result, 1)
```

Reply:
It seems better!
Fixed in https://github.com/access-company/antikythera/pull/168/commits/dcfca0aa83d5ccf4689ed66b9745f8d521d74f40